packages/web-components/src/components/va-card/va-card.tsx

```diff
@@ -23,15 +22,14 @@ export class VaCard {
    */
   @Prop() showShadow?: boolean = false;
 
-  render() {
-    const {
-      showShadow
-    } = this;
-
-    const classes = classnames('va-card',  {'show-shadow': showShadow});
+  /**
+   * If `true`, the card will have a gray background.
+   */
+  @Prop() background?: boolean = false;
 
+  render() {
     return (
-      <Host class={classes}>
```

Comment:
I think these classes were added originally to apply style to the host element but it's overcomplicating things. Style can be applied to the host using the `:host` selector and attributes like this instead:

```
:host([show-shadow]:not([show-shadow='false']))
```